xbmc/cores/VideoPlayer/Interface/TimingConstants.h

```diff
@@ -11,7 +11,7 @@
 #define DVD_TIME_BASE 1000000
 #define DVD_NOPTS_VALUE 0xFFF0000000000000
 
-constexpr int DVD_TIME_TO_MSEC(double x) { return static_cast<int>(x * 1000 / DVD_TIME_BASE); }
+constexpr long DVD_TIME_TO_MSEC(double x) { return static_cast<long>(x * 1000 / DVD_TIME_BASE); }
```

Comment:
imo this should better suit as `int64_t`

Reply:
okay, changed it.